content/en/docs/overview/singleton-sdk/Angular Client Introduction/data-initialization.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Data Initialization"
+date: 2019-09-24T20:08:31+08:00
+draft: false
+weight: 30
+---
+
+
+In general, for most of products, it's good enough to load all translations and patterns one time in root module when the application starts, but for some special requirements, such like: isolated translation for feature module or using lazy module to enhance performance, please refer to [Isolated feature module](https://ngx.eng.vmware.com/@vmw/ngx-vip/isolated-module/documentation) and [Lazy Load Module](https://ngx.eng.vmware.com/@vmw/ngx-vip/lazy-load/documentation). singleton client provides two ways to load i18n data and initialize the services in root module:
```

Comment:
The URI shouldn't link to internal ngx-component, it should redirect to isolated-feature-module.md

Reply:
I have fixed it.